paper-autocomplete.html

```diff
@@ -476,7 +485,8 @@
     _fireEvent: function (option, evt) {
       var id=this._getId();
       var event='autocomplete' + this.eventNamespace + evt;
-      this.fire(event,{id:id,value:option.value,text:option.text,target:this,option:option});
+      this.fire(event,{id:id,value:option[this.valueProperty] || option.value,text:option[this.textProperty] || option.text,target:this,option:option
```

Comment:
Not at all. When I was working on it, I had split it to multiple lines. But since you had it on one line, I put it back when I was done.